exercises/practice/perfect-numbers/.approaches/if-statements/content.md

```diff
@@ -77,6 +77,17 @@ return sumOfFactors < number ? Classification.Deficient :
        Classification.Perfect;
 ```
 
+## Combining Where and Sum vs. using Sum only
+Instead of using [Where][enumerable-where] and then [Sum][enumerable-sum] we could actually implement this by using [Sum][enumerable-sum] only.
```

Comment:
```suggestion

Instead of using [Where][enumerable-where] and then [Sum][enumerable-sum] we could actually implement this by using [Sum][enumerable-sum] only.
```